.github/workflows/release.yml

```
@@ -9,6 +9,8 @@ on:
       version:
         description: "Release version (e.g., 0.1.0)"
         required: true
+permissions:
+  contents: read
 
 permissions:
```

YAMLlint (1.37.1)

[error] duplication of key "permissions" in mapping

(key-duplicates)